README.md

```diff
@@ -1 +1,21 @@
-# hello-microverse
\ No newline at end of file
+# hello-microverse
+
+## This project is to help me develop a mastery over Github tools and best practices.
+
+# Primary Languages used in this project :
+
+HTML5
+CSS3
+
+## If you like my project PLEASE let me know bt leaving me a comment!
+
+👤 Author1
+
+GitHub: @andreacostabz
+Twitter: @andrebze
+LinkedIn: Andre Acosta
+
+Acknowledgments 
+
+Linters : Microverse
+Inspiration : Microverse 
```

Comment:
- [ ] To make the README file look more professional I would recommend following the Microverse [README template](https://github.com/microverseinc/readme-template), then customize it with your own information.

Reply:
Thank you! working on it.